spec/models/user_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+RSpec.describe User, :type => :model do
+  subject do
+    @user1 = FactoryBot.create(:user)
+  end
+
+  describe '#valid attributes' do
+    it 'is valid with valid attributes' do
+      subject
+      expect(@user1).to be_valid
+    end
+  end
+
+  describe '#unique mail, (uid,provider)' do
+    it 'has a unique email' do
+      subject
```

Comment:
why are u calling subject here  🤔 . Then you are using user2

Reply:
this is done to test uniqueness of email, i need a user1 created in order to 'not being able to create' user2